docs/spec/consensus/fork-accountability.md

```diff
@@ -0,0 +1,142 @@
+
+We consider three types of potential attack victims: 
+
+- full node
+- lite client with sequential header verification (LCS) and 
+- lite client with bisection based header verification (LCB).
+
+In case of full node, attack surface is constrained to the blockchain as a correct full node executes 
+all transactions so it can verify if computed application state diverge from the proposed state. Therefore,
+full node can be tricked to follow forked chain by committing different set of transactions to a blockchain.
+
+In case of lite clients, attack surface is bigger as lite client only verified headers without executing transactions.
+Therefore, faulty nodes can in some cases attack lite clients by arbitrarily changing state if there is a sufficient
+number of voting power that can commit such state. 
+
+Q: Is there an important difference between LCS and LCB, i.e., are there some attacks that are possible with LCB but not
+with LCS?
+
+### Equivocation based attacks
+
+In case of equivocation based attacks, faulty validators sign multiple votes (prevote and/or precommit) in the same
+round of some height. This attack can be executed on both full nodes and lite clients. It requires at least 1/3+ of voting power to be executed. 
+
+Example scenario: let's consider faulty proposer that propose block A to 1/3 of voting power equivalent correct processes (let's denote this set with C1) and proposes block B to 1/3 of voting power equivalent correct processes
+(lets denote this set with C2). Validators from the set C1 and C2 prevote for A and B respectively. Faulty validators
+that have +1/3 of voting power prevote both for A and B. Therefore correct validators from set C1 and C2 will observe 
++2/3 of prevotes for values A and B respectively and precommit for values A and B (respectively). Faulty processes 
+precomit for both values A and B, so we have commits for both values A and B. This is the attack on the full node level,
+that of course extends also to the lite clients, but it is easily detected at the main chain level as we assume that 
+correct nodes are able to eventually talk to each other. Creating an evidence of misbehavior is simple in this case
+as we have multiple messages signed by the same processes for different values in the same round. 
+
+In case of lite clients, a variant of this attack is possible where faulty activity is visible only to a lite clients.
+In this case +2/3 of voting power is needed. Note that once equivocation is used to attack lite client it opens space
+for different kind of attacks as application state can be diverged in any direction. For example it can modify validator set such that it contains only validators that don't have any stake bonded. In order to detect such (equivocation based attack) lite client need to cross check it's state with some correct validator (or to obtain
+hash of the state from the main chain using out of band channels). In case of equivocation attack client would 
```

Comment:
I don't quite understand. Shouldn't the light client only need to broadcast the signatures it has received (which would be equivocations vis-a-vis the canonical chain) to a full node, which can then perform the evidence submission?